app/webpacker/css/admin/products_v3.scss

```
@@ -21,13 +21,37 @@
 
   // Form actions floats over other controls when active
   .form-actions {
-    position: absolute;
-    top: -1em;
+    position: sticky;
+    top: 0;
     left: 0;
     right: 0;
     z-index: 1; // Ensure tom-select and .disabled-section are covered
   }
 
+  .form-actions:not(.hidden) + table.products {
+    thead {
+      top: 54px;
+
```

Clever hack to fill in the gap between with whitespace.  It would be good to add a comment to code like this to explain what it's for.
```suggestion

      // Fill in gap between .form-actions and thead with whitespace
```

Although I wish we didn't need such a hack (i notice it also covers up the drop shadow on .form-actions).
Perhaps .form-actions could be moved to a table row, but that might cause other problems..